components/Icon.tsx

```diff
@@ -39,6 +46,50 @@ const IconSvgs: Record<IconType, React.ReactElement> = {
       <path d="M7 1L7 13" strokeWidth="1.75" strokeLinecap="round" />
     </svg>
   ),
+  back: (
+    <svg
+      width="25"
+      height="12"
```

Comment:
Drop width and height here so they may be overridden by sizing class names on the Icon.
```suggestion
```

Reply:
@ethanlee16 When I removed this, the arrows disappeared, so just kept these lines in for now

Reply:
You may need to apply a specific width to the icon, i.e. `w-6` for it to get some natural width. It's not a huge deal, but it I believe this means that this icon doesn't have resizeable behavior (i.e., it will always be 25x12).

Reply:
sounds good! went ahead and moved the width and height out of the svg file like you suggested :) ty!!